### DISCUSSION TITLE
How to disable info messages in VSCode?

### DISCUSSION THREAD

**Discussion by anburocky3** [cmderdev/cmder#2692] [Q&A] [answered]
Every time, I open the terminal in vs code, I see this info text, it is not useful every time, I like to remove this info message, is there any way to remove this?

![image](https://user-images.githubusercontent.com/15264938/164256430-35f59359-1a13-4a54-87ed-e67b4815287e.png)

These are my vscode config btw:

```json
 "Cmder": {
      "path": [
        "${env:windir}\\Sysnative\\cmd.exe",
        "${env:windir}\\System32\\cmd.exe"
      ],
      "args": ["/K", "%CMDER_ROOT%\\vendor\\bin\\vscode_init.cmd"]
    }
```

**Comment by DRSDavidSoft** [comment, marked as answer]
Edit the `%CMDER_ROOT%\bin\vscode_init_args.cmd` file, navigate to the bottom and change **1** to **0** in both of the `set verbose_output=1` statements.

Example:
```cmd
if defined CMDER_CONFIGURED (
    rem Set Cmder settings here for when VSCode is launched inside Cmder.
    set verbose_output=0
) else (
    rem Set Cmder settings here for when VSCode is launched from outside Cmder.
    set verbose_output=0
)
```

Then, please also add a `rem` before both `echo Applying Cmder VSCode settings` lines in `vscode_init_args.cmd` as well:

```
rem echo Applying Cmder VSCode settings from '%~0'...
```

and

```
rem echo Applying Cmder VSCode settings is complete!
```

**Reply by anburocky3** [reply]
But still, i see this info: 
![image](https://user-images.githubusercontent.com/15264938/200338675-28636cc6-edac-4bde-aaf5-c5e3b5ac2381.png)

**Reply by DRSDavidSoft** [reply]
Please also add a `rem` before both `echo Applying Cmder VSCode settings` lines in `vscode_init_args.cmd` to remove these lines.

Nice color scheme, BTW!

**Reply by waptik** [reply]
Thanks. This still works like a charm
